frontend/components/surplus/CollectSurplusAuctionPanel.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+    <BasePanel :current-state="currentStateAndTitle.name">
+        <template #title>
+            {{ currentStateAndTitle.title }}
+            <time-till v-if="auction.state === 'have-bids'" :date="auction.earliestEndDate" />
+        </template>
+        <TextBlock v-if="auction.state === 'ready-for-collection'">
+            Since the end of the auction at {{ auction.earliestEndDate.toUTCString() }}, the winner can now collect the
+            auction amount.
+        </TextBlock>
+        <TextBlock v-else>
+            If no one else bids on the auction, the winner be able to collect the auction amount when the auction is
+            ended after {{ auction.earliestEndDate.toUTCString() }}
```

Comment:
Here I would be more general in the auction text and write: 
"If no one else bids on the auction, one will be able to collect the auction amount when the auction is ended after {{ auction.earliestEndDate.toUTCString()"
(background: basically everybody will be able to execute `deal()` function. Not only the winner